internal/pkg/template/workload.go

```diff
@@ -39,6 +39,12 @@ const (
 	DisablePublicIP         = "DISABLED"
 	PublicSubnetsPlacement  = "PublicSubnets"
 	PrivateSubnetsPlacement = "PrivateSubnets"
+
+	// RuntimePlatform configuration.
+	linuxOS = "LINUX"
+	// windowsCoreOS = "WINDOWS_SERVER_2019_CORE"
+	// windowsFullOS = "WINDOWS_SERVER_2019_FULL"
```

Comment:
Can we add these when we need them 🤔

Reply:
Sure. We'll use them soon, but I can take them out!